.gitignore

```diff
@@ -10,3 +10,4 @@ htmlcov
 .tox
 lib/createsend.egg-info
 .rake_tasks~
+.idea
```

Comment:
This is related to your PyCharm IDE and probably shouldn't be committed to master.

Reply:
It was in git ignore so that it wouldn't commit any of my IDE workspace changes to the branch. Should I still remove?

Reply:
The problem here is anyone else that uses an 'idea' IDE with this repo will have their workspace differences excluded too. There's merits for adding this, and for not. It's fine to leave it here. I'll leave the decision to you.

Reply:
I will remove it 👍

Reply:
We will *always* want the workspace data to be excluded from git history. This should be included in the `.gitignore`